patches/server/0803-Add-paper-mobcaps-and-paper-playermobcaps.patch

```diff
@@ -322,6 +322,7 @@ index bad8f0d2934218296954e9758810a68e1fedcacc..0b228b8bcae5ed2840ab244b6de35732
 +            case WATER_CREATURE -> world.getWorld().getWaterAnimalSpawnLimit();
 +            case AMBIENT -> world.getWorld().getAmbientSpawnLimit();
 +            case WATER_AMBIENT -> world.getWorld().getWaterAmbientSpawnLimit();
++            case UNDERGROUND_WATER_CREATURE -> world.paperConfig.perWorldSpawnLimits.get(net.minecraft.world.entity.MobCategory.MISC);
```

Comment:
MISC?

Reply:
that category never shows up here, look at the top of NaturalSpawner, it has a list of spawnable categories and MISC is filtered out. Which is why its filtered out in the PaperWorldConfig too

Reply:
oh nvm, I thought you were talking about smth else didnt scroll to the right on the line you highlighted